README.md

```diff
@@ -248,7 +248,7 @@ fun myFunction(dep: Dep, arg: String): String = ...
 
 ### Scopes
 
-By default kotlin-inject will create a new instance of a dependency each place it's injected. If you want to re-use an
+By default, kotlin-inject will create a new instance of a dependency each place it's injected. If you want to re-use an
```

Comment:
Same here

Reply:
ok, maybe I can revert it, I will only fix some typos :)